### PR TITLE
i18n(it): Updated every file at "root level"

### PR DESCRIPTION
**Pull request by casungo**
#### Description (required)

Updated `contribute.mdx`, `editor-setup.mdx`, `getting-started.mdx` and `recipes.mdx`

### CODE REVIEW

**Comment by casungo** [comment]
The failed check is a broken link check. It complaints about `/it/community-resources/content/#ricette-e-guide` not being there. #6836 fixes this because I translated that also :)

File: src/content/docs/it/recipes.mdx
Comment (dreyfus92) [inline src/content/docs/it/recipes.mdx:20]:
```suggestion
<ReadMore>Guarda ancora più ricette scritte e mantenute dalla community nelle nostre [ricette e guide della community](/it/community-resources/content/#recipes-and-guides)</ReadMore>
```

**Review by dreyfus92 — COMMENTED** [review]
Hello @casungo, this suggestion should fix error thrown by the pipeline 😁

File: src/content/docs/it/recipes.mdx
Comment (casungo) [reply src/content/docs/it/recipes.mdx:20]: As I said in [this comment](https://github.com/withastro/docs/pull/6835#issuecomment-1934819625) the broken link will be resolved once the mentioned PR I made is merged. The changes are separated because they were in a different folder and I created 3 different PRs to keep everything organized instead of having a big PR. If I committed this change the link would be broken in the main branch. Hope this clears everything :)

Comment (dreyfus92) [reply src/content/docs/it/recipes.mdx:20]: Got it, thanks for breaking that down for me!

**Review by delucis — APPROVED** [review]
Thanks @casungo — looks good to me and exciting to see the new landing page implemented in Italian 🎉 

Will still need to wait on #6836 to merge before we merge this to fix the broken link.

**Pull request merged**
